depobs/website/templates/statistics.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div>
+        <h1 class="mb-4">Statistics</h1>
+    </div>
+
+    <div id="statistics">
+        <h2 class="h3 border-bottom border-light mb-3 pb-2" id="faq-general">Distribution of letter scores</h2>
+        <img src ="/histogram.png">
```

Comment:
nit: the space after the `src =` is kinda weird; could be this diff view / zoom level